omnisharp.json

```diff
@@ -4,5 +4,11 @@
      },
      "cake": {
          "enabled": false
+     },
+     "dotnet": {
+         "enabled": false
```

Comment:
> Also updated the omnisharp.json to not attempt to provide C# completion for Razor.VSCode since KoreBuild doesn't work well in an O# world.

I don't get it. What is this for?

Reply:
If you have VSCode open in the Razor.VSCode folder you get a bajillion C# errors about things not being found. Essentially it stops you from running the extension.